Lib/test/_test_multiprocessing.py

```diff
@@ -782,6 +782,13 @@ def test_sys_exit(self):
             join_process(p)
             self.assertEqual(p.exitcode, reason)
 
+        for args in ((None,), (),):
```

Comment:
I would like add an integer here

Reply:
The test for the integer case should be covered earlier in the test case [here](https://github.com/python/cpython/pull/11538/files/36f0c9bc1990cc9507626f96f71caade59e274b8#diff-b046ab474480855fb4a01de88cfc82bbR778). I can combine both sets of cases.